### PR TITLE
Add http rbs

### PR DESCRIPTION
**Pull request by sue445**
I add rbs for https://github.com/httprb/http

### CODE REVIEW

File: gems/http/5.1/http.rbs
Comment (ybiquitous) [inline gems/http/5.1/http.rbs:6]: [suggest] I'm unsure but the built-in `_ToS` interface may be available instead:

https://github.com/ruby/rbs/blob/v3.1.1/core/builtin.rbs#L13-L15

File: gems/http/5.1/_test/test.rb
Comment (ybiquitous) [inline gems/http/5.1/_test/test.rb:3]: [suggest] It's possible to remove the scaffold comment.

```suggestion
```

File: gems/http/5.1/http.rbs
Comment (ybiquitous) [inline gems/http/5.1/http.rbs:2]: [suggest] It's possible to remove the scaffold comment, too.

```suggestion
```

File: gems/http/5.1/http.rbs
Comment (ybiquitous) [inline gems/http/5.1/http.rbs:39]: [suggest] This internal comment is shown as a public doc on IDE, so it seems better to remove it.

```suggestion
```

**Review by ybiquitous — COMMENTED** [review]
@sue445 Thanks for the Pull Request! I've commented on a few points of a bit concern.

File: gems/http/5.1/http.rbs
Comment (sue445) [reply gems/http/5.1/http.rbs]: @ybiquitous Thank you! I fixed at 3b6d7cb

**Review by ybiquitous — COMMENTED** [review]
LGTM! But I recommend getting other maintainers' approval.

**Comment by sue445** [comment]
@pocke Could you review this patch?

**Review by pocke — APPROVED** [review]

**Pull request merged**
